code/src/components/ProgressBar.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { useSelector } from "react-redux";
+
+export const ProgressBar = () => {
+    const todos = useSelector(store => store.todos);
+    const total = todos.length;
+
+    if (total > 0) {
+        const completed = todos.filter(todo => todo.isCompleted).length;
+
+        return (
+            <div className="ProgressBar">
+                {completed} of {total} done
+      </div>
```

Comment:
nice to only show the 1 of 3 header ( or whatever the tasks you have), when you actually have any tasks in the list.
Keep an eye on you indentation, tho. looks like you have tabsize 4 instead of 2?